debian/changelog

```diff
@@ -1,3 +1,12 @@
+kora-util (1.2.0) unstable; urgency=low
+
+  * feature(dynamic): make object_t::at method and rvalues friends.
+    This patch allows to use rvalues as a default parameter for `at`
+    method. It saves from the common mistake, where you want to obtain
+    reference to the object in map if it exists, or default value otherwise.
+
+ -- Evgeny Safronov <division494@gmail.com>  Fri, 02 Oct 2015 12:20:46 +0300
+
```

Comment:
It would be good to remove this commit from the PR to devote conversation for the new feature only and make another one PR with version bump. That PR could have a description with the list of all new features.